### DISCUSSION TITLE
Help Needed: Removing Unused Keys with Tolgee CLI in Projects with Dynamically Generated Tags

### DISCUSSION THREAD

**Discussion by kuji2336** [tolgee/tolgee-platform#2403] [Q&A]
Hi everyone,

I've been using Tolgee's self-hosted solution for a while in my project, and it has been working great overall. However, I've encountered an issue that I could use some help with.

Over time, my project has accumulated a lot of unused keyNames that are no longer used in the application. I know that I can delete unused keys using the CLI command with the --remove-unused flag, which is a great feature. However, my project mostly consists of dynamically generated tags, and it seems that the CLI command cannot detect these dynamically generated keys.

Has anyone else faced this issue or found a workaround for removing unused keys in projects with dynamically generated tags? Any tips or guidance would be greatly appreciated.
<img width="733" alt="image" src="https://github.com/tolgee/tolgee-platform/assets/58769030/6542a175-ea93-4a0d-9548-3659459888ce">

**Comment by stepan662** [comment]
Hey, with dynamic keys it's hard to do.
